### PR TITLE
Python requests for TimeRange feature, branch pointing to release/6.6.0. Addressed all code review comments provided in https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/3.

### PR DESCRIPTION
**Pull request by Arunima19**

### CODE REVIEW

**Comment by Arunima19** [comment]
@stleary-cisco Addressed all the review comments from https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/3. Facing some issues in https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/3. Closing the old PR (https://github.com/stleary-cisco/ftd-rest-api-cookbook/pull/3)

File: examples/ftd_api_scripts/timerange-access-rule-association.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+        print('Unable to update access rule')
+        return False
+    print('Access rule name is {}'.format(updated_access_rule['name']))
+    return True
+
+
+if __name__ == '__main__':
+
+    if len(sys.argv) != 6:
+        print("Usage: python cookbook_scripts/timerange-access-rule-association.py host port user passwd")
```
Comment (stleary-cisco) [inline examples/ftd_api_scripts/timerange-access-rule-association.py]: parentId is a required param but is not shown in usage. Similar problem in timerange_access_rule_association_actions() where the parentid param is not included in docstring. But isn't the parentId for access policy hardcoded to 'default'? Why is the user required to pass it into main/timerange_access_rule_association_actions()? Should it just be hardcoded in timerange_access_rule_association_actions() ?

Comment (stleary-cisco) [inline examples/ftd_api_scripts/timerange-access-rule-association.py]: cookbook_scripts-> ftd_api_scripts

Comment (Arunima19) [reply examples/ftd_api_scripts/timerange-access-rule-association.py]: Done

Comment (Arunima19) [reply examples/ftd_api_scripts/timerange-access-rule-association.py]: Done

**Review by jaredtsmith — APPROVED** [review]

**Review by stleary-cisco — APPROVED** [review]

**Pull request merged**